config/serverless.yml

```diff
@@ -48,7 +48,7 @@ data.search.sessions.enabled: false
 advanced_settings.enabled: false
 
 # Enforce restring access to internal APIs see https://github.com/elastic/kibana/issues/151940
-# server.restrictInternalApis: true
+server.restrictInternalApis: true
```

Comment:
Hum, Tina or JL would need to confirm, but I think doing that in the "production" config file may break a lot of things ATM.

Why not doing it via CLI args in the FTR suites that are supposed to run against serverless?

More globally, I don't think a test-only feature or changes should **ever** need to touch the `config` folder.

Reply:
@jloleysens I missed this.

Reply:
Yeah, we are working through this on this PR https://github.com/elastic/kibana/pull/162636.

We plan on turning this on here in ~1 week https://github.com/elastic/kibana/pull/162149